google-cloud-tools-plugin/src/com/google/gct/idea/appengine/cloud/AppEngineDeploymentRunConfigurationEditor.java

```diff
@@ -102,7 +102,7 @@ public void actionPerformed(ActionEvent e) {
       }
     });
     userSpecifiedArtifactFileSelector.addBrowseFolderListener(
-        GctBundle.message("appengine.managedvm.config.user.specified.artifact.title"),
+        GctBundle.message("appengine.flex.config.user.specified.artifact.title"),
```

Comment:
This class is renamed have 'AppEngine', but is using a property named 'flex'. Isn't this class Flex specific?